src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -162,24 +151,17 @@ public WinHttpHandler()
         }
 
         #region Properties
-        public AutomaticRedirectionPolicy AutomaticRedirectionPolicy
+        public bool AutomaticRedirection
```

Comment:
@stephentoub Yes, the intent is to change the public API.  This change is per security review guidance.